src/common/schemas/PatchBucketRequest.yml

```diff
@@ -1,9 +1,19 @@
 type: object
-description: Updates to an existing bucket resource.
+description: |
+  Retention rules to expire or retain data.
+  #### InfluxDB Cloud
+  - `retentionRules` is required.
+
+  #### InfluxDB OSS
+  - `retentionRules` isn't required.
```

Comment:
- Can't it also update `name` and `description`?g
- Should we copy-paste the other description of `retentionRules` here?

Reply:
you're right, it can update `name` and `description`. I placed this description in the wrong place. I had meant to put it under the description for `retentionRules` itself and not for the request description. thanks for catching this.

Reply:
Changing this description to be a description of the request